src/ui/server/constants/declarations/versions/index.ts

```diff
@@ -16,7 +16,7 @@ const DECLARATION_VERSIONS = [
     HOW_YOUR_DATA_WILL_BE_USED: '1',
   },
   {
-    ANTI_BRIBERY: '1',
+    ANTI_BRIBERY: '2',
```

Comment:
I've added a unit test for this file in https://github.com/UK-Export-Finance/exip/pull/2819 

Happy to merge either first